src/ui/i18n.ts

```diff
@@ -54,12 +79,25 @@ export const translations = {
     'Automatic quality': 'Automatisch',
     'Enable subtitles': 'Untertitel an',
     'Disable subtitles': 'Untertitel aus',
+    'Rewind {s} seconds': 'Rückspulen {s} sekunden',
+    'Forward {s} seconds': 'Vorwärts bewegen {s} sekunden'
   },
 };
 
-export const getTranslation = languageCode => text => {
-  if (translations[languageCode] && translations[languageCode][text]) {
-    return translations[languageCode][text];
+let translationsMerge = false
+
+export const getTranslation = (languageCode, translationsUi) => (text, params?: object) => {
+  if(!translationsMerge) {
```

Comment:
I'd rather have this based on an sprintf(_s) package instead of maintaining this ourselves. This way, we would probably reduce the code to a single line.